docs/API/InstrumentFormat.md

```diff
@@ -321,7 +532,11 @@ is NOT REQUIRED and unused.
 
 
 ### 2.2.1: HeaderElement
-
+```
+Not implemented
+Title is rendered from the meta LongName
+ZV
```

Comment:
This is both currently implemented by the API and required by LORIS, it's for headers inside of the instrument , not the instrument name.